.mvn/maven.config

```diff
@@ -0,0 +1,6 @@
+-V
```

Comment:
Super neat! https://github.com/elastic/apm-agent-java/blob/974ca32266fefb599d5560e93f4dbdce349473fd/.github/workflows/snapshoty.yml#L15-L21 is using the old fashion way, would you mind to update it too? 🙏